spec/services/rapid_ready_for_decision/hypertension_upload_manager_spec.rb

```diff
@@ -26,13 +26,24 @@
     )
   end
 
+  let(:time_freeze_time) { '10-10-2021' }
+
+  before do
+    Timecop.freeze(time_freeze_time)
+  end
+
+  after do
+    Timecop.return
+  end
```

Comment:
Borrowed from in order to have a stable date to assert with: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/requests/backend_status_request_spec.rb

Reply:
@yoomlam do you have any issue with this approach to testing this? Is it overkill? I know testing with dates can be a source of flakiness and wanted to avoid it!

Reply:
You should use a time zone whenever creating a timestamp (details: https://thoughtbot.com/blog/its-about-time-zones) to ensure your test setup is using the same zone as your implementation.
Also all the [examples](https://github.com/travisjeffery/timecop) I've seen for `Timecop.freeze` passes in a Time object rather than a string.

Reply:
Yep, this is lifted from other places in the code but we should improve the approach. I'll update!